### DISCUSSION TITLE
Looking for Maintainer

### DISCUSSION THREAD

**Discussion by burhanrashid52** [burhanrashid52/PhotoEditor#314] [General]
Hello All,

3 years back I open-source and this library which was inspired by [PhotoEditorSDK ](https://github.com/eventtus/photo-editor-android)and[ Imgly Android App](https://play.google.com/store/apps/details?id=com.photoeditorsdk.android.app&hl=en_IN&gl=US). Since then I am maintaining this project solely in my free time. Along the way, I found few collaborators who help me to fix minor and major bug fixes (Thank you for the fixes :)).

Since it's been used by a lot of developers with around 200 thousand downloads, a lot of issues, questions are being raised on the repo.

Currently, I am not able to find an ample amount of time to maintain this project. So now I am looking for a maintainer/collaborator for this project. If anyone who is interested can start the discussion here.

**Comment by isagardb** [comment]
Yes im interested

**Reply by burhanrashid52** [reply]
I've added the list in the below [comment](https://github.com/burhanrashid52/PhotoEditor/discussions/314#discussioncomment-380647). Let me know what you can pick.

**Comment by lucianocheng** [comment]
Happy to help with reviewing bug fixes when possible.ᐧ
ᐧ

On Wed, Feb 17, 2021 at 8:16 PM Sagar D B <notifications@github.com> wrote:

> Yes im interested
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/burhanrashid52/PhotoEditor/discussions/314#discussioncomment-379886>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAH2GCSKQN4CLVAK4XQCGK3S7SIDLANCNFSM4XYE4GMQ>
> .
>


-- 
Luciano Cheng
Cell: (609) 577-1065
Twitter: @lucianocheng <http://www.twitter.com/lucianocheng>
Linkedin: http://www.linkedin.com/in/lucianocheng

**Reply by burhanrashid52** [reply]
Were you able to find any issue to work on?

**Comment by burhanrashid52** [comment]
Thank you. We can start with
1. Resolve question issues and encourage them to move it to the discussion, if it's not a bug or feature enhancement. This kind of issues are mostly "How to do X" and"Where I can find X" like #313 #304 #298 
2. Fix bugs like #291 #249 #232. We need an issue template. I don't have one right now. Any suggestions are welcome.
3. Improve sample app #316 #277 #226
4. Improve documentation. How to crop using effectfactory.

**Comment by basitis** [comment]
Let's start

**Reply by burhanrashid52** [reply]
So what issue are you planning to pickup?

**Reply by burhanrashid52** [reply]
Were you able to find any issue to work on?

**Reply by basitis** [reply]
Let me start with [#313](https://github.com/burhanrashid52/PhotoEditor/issues/313)

**Comment by burhanrashid52** [comment]
I've added a stale bot to close the issue which is inactive for 14 days.

**Reply by basitis** [reply]
Can I help with mark issues with labels @burhanrashid52 ?

**Reply by burhanrashid52** [reply]
Yeah sure. Go ahead.
